Update google credential generator doc (#214)
ResultadosDigitais/matrix: README.md

If you want run the **#matrix**, you need follow steps:

1. Clone this repository `git clone git@github.com:ResultadosDigitais/matrix.git`

2. We are using Google to authorizations, you need create a credential [here](/docs/GOOGLE-CREDENTIAL-STEP-BY-STEP.md) you can follow step by step

> 3. in the root create a file variables.env

Author: ```suggestion
3. in the root create a file `variables.env`
```

> 3. in the root create a file variables.env

Author: Done!

		GOOGLE_CLIENT_ID=000000000-xxxxxxxxxx.apps.googleusercontent.com

> 		GOOGLE_SECRET=gXXXXXXXXXXXXss

Reviewer: O que acha de deixar só o `GOOGLE_CLIENT_ID` e `GOOGLE_SECRET` para ficar mais simples? Os outros já estão como valor padrão no config. Aí você poderia colocar um link para o config para ele olhar e ver se quer alterar algum valor padrão.

Author: O ruim é que nem todo mundo manja ou tem afinidade com código fonte.. apesar de estar bem simples..

Reviewer 2: Eu não me lembro agora exatamente o termo, mas seria bom deixar um nome proximo do que o Google usa.

Reviewer: @dukex  você está falando do nome da variável? "client id" e "secret" é o nome que a Google usa:

![google-credential-step-6](https://user-images.githubusercontent.com/673904/77076102-1aafd100-69d2-11ea-879d-ceb2d5d96a02.png)

Reviewer: @juliemar  faz sentido deixar então.

Reviewer 2: Cheguei meio atrasado, mais é isso mesmo

		GOOGLE_CALLBACK_URL=http://localhost:8080/auth/google/callback
		COOKIE_SESSION_SECRET=matrix-session
		COOKIE_SESSION_MAX_AGE=2592000000
		ENFORCE_SSL=false

> 		WHITELIST_DOMAINS=[]

Reviewer: `WHITELIST_DOMAINS="[]"`

Author: Done!

> 		WHITELIST_DOMAINS=[]

Author: ```suggestion
		WHITELIST_DOMAINS="[]"
```

The **#matrix** project has some environments that important to define.

1. We are using Google to authorizations, you need create a credential [here](https://developers.google.com/identity/sign-in/web/sign-in) and before define this variables:

		GOOGLE_CLIENT_ID=${paste_your_client_id_here}
		GOOGLE_SECRET={paste_your_secret_here}
		GOOGLE_CALLBACK_URL=http://localhost:8080/auth/google/callback

> 		COOKIE_SESSION_SECRET=matrix-session

Author: ```suggestion
```

> 		COOKIE_SESSION_MAX_AGE=2592000000

Reviewer: `COOKIE_SESSION_SECRET` e `COOKIE_SESSION_MAX_AGE` não são variáveis da autorização do Google, são da sessão do app. Agora com o passport podemos colocar Facebook, Github, login e senha, etc, e a sessão vai ser única independente do provedor de autenticação.

Author: vou remover daqui

> 		COOKIE_SESSION_MAX_AGE=2592000000

Author: ```suggestion
```

2. You can define a white List of trusted email domains can enter in the **#matrix**

> 		WHITELIST_DOMAINS=["@domain1.com","@domain2.com"]

Reviewer: Lá no JavaScript dá para colocar o array direto, mas na variável de ambiente tem que estar como string. Chegou a testar se assim funciona?

Author: testei e no arquivo `variables.env`foi de boa.. no spinaker tbm

Reviewer: Então show! Melhor ainda!

Reviewer: Eu tinha falado para colocar `WHITELIST_DOMAINS="[]"` em um comentário que já marquei como resolvido, então lá pode ser como estava antes: `WHITELIST_DOMAINS=[]`

### External Meet
The default video conferencing in meetings is [Jitsi](https://jitsi.org/jitsi-meet/), but you can change that in any room, using [Meet](https://meet.google.com/) or [Zoom](https://zoom.us/). For that, you just need provide the parameter `externalMeetUrl` in your room config:

		ROOMS_DATA=[
		   {
		      "id":"${UUID}",
		      "name":"Meeting External",
			  "externalMeetUrl": "https://external-url-room/key-room"
		   }
		 ]

# Running in Production

> If you will run in production we strongly recomend to you close your environmen using an internal VPN. In this solution everybody with the link and a valid google credential can enter in your virtual office. Because of this is important to you mantain your environment closed. Or you can define a variable `WHITELIST_DOMAINS` to limit only authorized users enter in the **#matrix**. 

Reviewer: Seria bom reforçar que deveriam mudar o valor padrão do 

- `COOKIE_SESSION_SECRET` para uma chave forte própria.
- `GOOGLE_CALLBACK_URL` para o domínio da URL que vai usar

Author: Done!

> If you will run in production we strongly recomend to you close your environmen using an internal VPN. In this solution everybody with the link and a valid google credential can enter in your virtual office. Because of this is important to you mantain your environment closed. Or you can define a variable `WHITELIST_DOMAINS` to limit only authorized users enter in the **#matrix**. 

Author: ```suggestion
If you will run in production we strongly recommend you close your environment using an internal VPN. In this solution everybody with the link and a valid google credential can enter your virtual office. Because this is important for you to maintain your environment closed. Or you can define a variable `WHITELIST_DOMAINS` to limit only authorized users to enter in the **#matrix**. You have to choose a strong key to the `COOKIE_SESSION_SECRET` and have to put in `GOOGLE_CALLBACK_URL` your production domain.
```
